project.godot

```diff
@@ -177,11 +178,15 @@ manipulator_axis_5={
 "events": [Object(InputEventJoypadButton,"resource_local_to_scene":false,"resource_name":"","device":-1,"button_index":9,"pressure":0.0,"pressed":true,"script":null)
 ]
 }
-manipulator_gripper={
+manipulator_axis_6={
 "deadzone": 0.5,
 "events": [Object(InputEventJoypadButton,"resource_local_to_scene":false,"resource_name":"","device":-1,"button_index":10,"pressure":0.0,"pressed":true,"script":null)
 ]
 }
+manipulator_gripper={
```

Comment:
What's the purpose?

Reply:
Adressed in [c7e1660](https://github.com/wisniax/RoverControlApp/pull/32/commits/c7e16602d1c93990e55a49808a1da0aec6af20b4)